config_defaults.yml

```diff
@@ -609,6 +609,7 @@ Gemfile:
     # beaker: true
 spec/default_facts.yml:
   ipaddress: "172.16.254.254"
+  ipaddress6: "FE80:0000:0000:0000:AAAA:AAAA:AAAA"
```

Comment:
Note - As per https://tools.ietf.org/html/rfc4291#section-2.4  FE80::/10 is link local

Also note that the IP4 address of 172... is not link local but private use.
MAC is a local administered address (i.e. not a vendor specified MAC)